packages/colorpickers/stories/examples/ColorDialog/WithFormInput.tsx

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Story, Meta } from '@storybook/react';
+import { Field, Label, Input, InputGroup } from '@zendeskgarden/react-forms';
+import { IColor, ColorDialog } from '@zendeskgarden/react-colorpickers';
+
+export default {
+  title: 'Components/ColorDialog',
+  component: ColorDialog
+} as Meta;
+
+const validHex = /^#(?<hex>[0-9A-F]{3}){1,2}$/iu;
+
+const StyledField = styled(Field)`
+  margin: auto;
+  width: ${props => props.theme.space.base * 41.25}px;
+`;
+
+const StyledInput = styled(Input)`
+  width: ${props => props.theme.space.base * 23.25}px;
+`;
+
+const StyledColorDialog = styled(ColorDialog)`
+  padding: 0 ${props => props.theme.space.base * 8}px;
+`;
+
+const StyledContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-top: ${props => props.theme.space.base}px;
+`;
+
+export const WithFormInput: Story = ({
+  placement,
+  alphaSlider,
+  hueSlider,
+  hex,
+  red,
+  green,
+  blue,
+  alpha,
+  disabled,
+  hasArrow,
+  isAnimated
+}) => {
+  const labels = { alphaSlider, hueSlider, hex, red, green, blue, alpha };
+  const [input, setInput] = useState('#17494d');
+  const [color, setColor] = useState<string | IColor>('rgba(23, 73, 77, 100)');
+
+  return (
+    <StyledField>
+      <Label>Favorite color</Label>
+      <StyledContainer>
+        <InputGroup>
```

Comment:
I'm glad we pushed this boundary by digging into the system more deeply. We find that border radius styling is leaking into the RGBA fields. Those fields will need specificity overrides to prevent this from happening.

<img width="426" alt="Screen Shot 2021-02-23 at 11 40 49 AM" src="https://user-images.githubusercontent.com/143773/108898495-1faad680-75cc-11eb-83b4-1dc7c70efd10.png">

Also, check with @steue, but the default placement on this example should _probably_ be `bottom-end`.

Reply:
Also note that the alpha value is showing up as `1000` in this example.

Reply:
Added overrides to prevent those UI issues and works across all browsers except IE11.

<img width="332" alt="Screen Shot 2021-02-26 at 10 41 41 AM" src="https://user-images.githubusercontent.com/1811365/109341771-b9b68d00-781f-11eb-800a-09c9a2e88116.png">

Reply:
Update: I've pushed a commit that fixes this in IE11 by refactoring the styling and using a `span` within the `botton` element.